skins/oasis/modules/ContentDisplayController.class.php

```diff
@@ -26,34 +26,34 @@ static function onShowEditLink(&$parser, &$showEditLink) {
 	 */
 	public static function onDoEditSectionLink( $skin, Title $title, $section, $tooltip, &$result, $lang = false ) {
 		global $wgBlankImgUrl, $wgUser;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgUser" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)